### PR TITLE
Login, Logout controllers implemented

### PR DESCRIPTION
**Pull request by hpavlo**

### CODE REVIEW

File: src/main/java/taxi/controller/LoginController.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/login.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+
```
Comment (shmex-j) [inline src/main/java/taxi/controller/LoginController.java:29]: Empty line

File: src/main/java/taxi/web/filter/AuthenticationFilter.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        allowedUrls.add("/drivers/add");
+    }
+
+    @Override
+    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse,
+                         FilterChain filterChain) throws IOException, ServletException {
+        HttpServletRequest req = (HttpServletRequest) servletRequest;
+        HttpServletResponse resp = (HttpServletResponse) servletResponse;
+        HttpSession session = req.getSession();
+
```
Comment (shmex-j) [inline src/main/java/taxi/web/filter/AuthenticationFilter.java:29]: Empty line

**Review by shmex-j — CHANGES_REQUESTED** [review]
Great job! Just few empty lines

**Review by ChumakAn — APPROVED** [review]
looks good to me 👍

**Pull request closed without merging**